src/com/oltpbenchmark/benchmarks/tpcc/ddls/tpcc-clickhouse-ddl.sql

```diff
@@ -0,0 +1,163 @@
+-- TODO: c_since ON UPDATE CURRENT_TIMESTAMP,
+
+DROP TABLE IF EXISTS TPCC.ORDER_LINE;
+CREATE TABLE TPCC.ORDER_LINE (
+  OL_W_ID int,
+  OL_D_ID int,
+  OL_O_ID int,
+  OL_NUMBER int,
+  OL_I_ID int,
+  OL_DELIVERY_D DateTime,
+  OL_AMOUNT Float32,
+  OL_SUPPLY_W_ID int,
+  OL_QUANTITY Float32,
+  OL_DIST_INFO FixedString(24),
+  EventDate Date DEFAULT toDate(now()),
```

Comment:
Please make the column names match the case of the other columns in the table. Also, add comments that explain what this does and why it is needed.